sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -937,6 +937,15 @@ object SQLConf {
       .timeConf(TimeUnit.SECONDS)
       .createWithDefault(0L)
 
+  val THRIFTSERVER_FORCE_CANCEL =
+    buildConf("spark.sql.thriftServer.forceCancel")
```

Comment:
thriftServer? thriftserver? Seems only `spark.sql.thriftserver.ui.retainedSessions` uses `thriftserver`.

Reply:
Seems recent config names `thriftServer`. `thriftserver` is early version config.